src/components/giving-tuesday/Countdown.js

```diff
@@ -1,10 +1,13 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import classNames from 'classnames';
 import SbEditable from 'storyblok-react';
 
 import CountdownPie from './CountdownPie';
 import UseCountdown from '../../hooks/useCountdown';
 
+const generateClassName = (key) =>
+  `${key}-${Math.floor(Math.random() * 100000, 5)}`;
```

Comment:
If you are worried about uniqueness you could use the built-in hook https://react.dev/reference/react/useId

Reply:
i will switch it up to `useId`

Reply:
actually it looks like `useId` was introduced in React18, and the site is still on 17.

Reply:
Oh huh, I thought we moved all our stuff to 18 already. No biggie. 

Just as a FYI, at some point, we will be replacing Gatsby for Next.js in each of our sites including this one. Just thought I would give you the heads up but nothing has started for this project yet and it will likely be some time before it does.

Reply:
sounds good, in my brief experience working on Next it felt similar to Gatsby. what's the reason for the switch, is Gatsby deprecated?

Reply:
The TLDR; is that Netlify bought Gatsby (the company) and isn't making the same investment into the JS Framework. 

Gatsby is on the decline
https://2022.stateofjs.com/en-US/libraries/rendering-frameworks/

and we are using Next in other places and like it.